e2e/test/scenarios/dashboard/reproductions/13736-dashboard-should-work-if-some-cards-are-broken.cy.spec.js

```diff
@@ -0,0 +1,48 @@
+import {
+  getDashboardCards,
+  restore,
+  updateDashboardCards,
+  visitDashboard,
+} from "e2e/support/helpers";
+
+describe("issue 13736", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should work even if some cards are broken (metabase#13736)", () => {
+    cy.intercept("POST", "/api/dashboard/*/dashcard/*/card/1/query", {
+      statusCode: 500,
+      body: {
+        cause: "some error",
+        data: {},
+        message: "some error",
+      },
+    });
+
+    cy.createDashboard({ name: "13736 Dashboard" }).then(
+      ({ body: { id: dashboardId } }) => {
+        updateDashboardCards({
+          dashboard_id: dashboardId,
+          cards: [
+            {
+              card_id: 1,
+            },
+            {
+              card_id: 2,
+              col: 11,
```

Comment:
Just so the dashboard cards are positioned next to each other nicely
![Screenshot 2023-08-03 at 10 11 31 PM](https://github.com/metabase/metabase/assets/1937582/b3269613-547f-4c48-8b88-8a4f8fc316b5)